homeassistant/components/netgear/sensor.py

```diff
@@ -68,39 +68,31 @@ class NetgearSensorEntityDescription(SensorEntityDescription):
         entity_category=EntityCategory.DIAGNOSTIC,
         native_unit_of_measurement=DATA_MEGABYTES,
         icon="mdi:upload",
-        value=lambda router, key: router.traffic_data.get(key)
-        if router.traffic_data is not None
-        else None,
+        value=lambda data: data,
```

Comment:
I'd make this lambda the default lambda in the description. Then we don't need a mixin and we don't need to specify the lambda in all descriptions.

Reply:
hahaha, just done exactly that and now reading this...